assets/sass/components/reader-revenue-manager/_googlesitekit-rrm-setup-module.scss

```diff
@@ -27,6 +27,13 @@
 		.googlesitekit-publication-onboarding-state-notice {
 			margin-bottom: 40px;
 		}
+
+		&.googlesitekit-setup-module-edit {
+
+			.googlesitekit-publication-onboarding-state-notice {
+				margin-top: 16px;
+			}
+		}
```

Comment:
This is no longer needed anymore as the publication select now has horizontal spacing around it. Instead, we need some top spacing for this in the settings view screen.